meson.build

```diff
@@ -39,44 +41,55 @@ dwave_optimization_src = [
 if host_machine.system() == 'windows'
     # Shared libraries don't work so well on windows, so we build from a static
     # library instead
-    libdwave_optimization = static_library(
-        'dwave-optimization',
-        sources: dwave_optimization_src,
-        include_directories: dwave_optimization_include,
+    libdwave_optimization = declare_dependency(
```

Comment:
`declare_dependency` just packages the library with the include for subsequent commands, so a bit tidier.